src/main/resources/application.yaml

```diff
@@ -25,6 +25,10 @@ stargate:
   exception-mappers:
     enabled: false
 
+  feature: # See io.stargate.sgv2.jsonapi.config.feature.DataApiFeatureConfig
```

Comment:
not: can we expand the docs here, or point to expanded docs on the DataApiFeatures class to explain the logic for setting values.

Reply:
Fixed link wrt name, but thinking of documenting only/mostly on code side.